Discovery/Web-Content/uri-from-top-55-most-popular-apps.txt

```diff
@@ -0,0 +1,2418 @@
+delivery/restaurants-near-me
+details/json
+device/keyvalue
+device/login
+device/login_status
+device/share
+devtools.html
+digester-rules.xml
+direct_messages.json
+direct_messages.json?full_text=true
+direct_messages/destroy.json?id=
+direct_messages/new.json
+direct_messages/sent.json
+direct_messages/sent.json?full_text=true
+direct_messages/show.json?id=
+domain_prod.json
+ds-test-ec.txt
+ds-test-rsa.txt
+ecodrive/blueandme_firmware.zip
+ecodrive/ecodrive_endpoints_env_dev1.json
+ecodrive/ecodrive_endpoints_env_dev2.json
+ecodrive/ecodrive_endpoints_env_int1.json
+ecodrive/ecodrive_endpoints_env_intpi.json
+ecodrive/ecodrive_endpoints_env_mock.json
+ecodrive/ecodrive_endpoints_env_preprod.json
+ecodrive/ecodrive_endpoints_env_prod.json
+ecodrive/ecodrive_endpoints_env_sit1.json
+ecodrive/ecodrive_endpoints_env_sit2.json
+ecodrive/ecodrive_endpoints_env_sys1.json
+ecodrive/ecodrive_endpoints_env_sys2.json
+ecodrive/ecodrive_logo.png
+ecodrive/market
+ecodrive/registration/ecodrive_new.css
+ecodrive_endpoints_env_openam.json
+eligible/payment-methods
+eligible/product/savings
+endpoints_env_dev1.json
+endpoints_env_dev2.json
+endpoints_env_int1.json
+endpoints_env_intpi.json
+endpoints_env_mock.json
+endpoints_env_openam.json
+endpoints_env_preprod.json
+endpoints_env_prod.json
+endpoints_env_sit1.json
+endpoints_env_sit2.json
+endpoints_env_sys1.json
+endpoints_env_sys2.json
+entries.txt
+error/NonExistentClass
+espresso/carts
+eth/pushtx
+events/publish
+f\u0014cek\u0018k_beprdrff
+favorites/create.json?id=
+favorites/destroy.json?id=
+favorites/list.json
+favorites/list.json?user_id=
+findplacefromuserlocation/json
+flutter/accessibility
+flutter/isolate
+flutter/keyevent
+flutter/lifecycle
+flutter/localization
+flutter/mousecursor
+flutter/navigation
+flutter/platform
+flutter/platform_views
+flutter/restoration
+flutter/settings
+flutter/system
+flutter/textinput
+followers/ids.json
+followers/ids.json?cursor=
+followers/ids.json?user_id=
+followers/list.json
+followers/list.json?user_id=
+fonts/MaterialIcons-Regular.otf
+fonts/OpenSans-Bold.ttf
+fonts/OpenSans-Italic.ttf
+fonts/OpenSans-Light.ttf
+fonts/OpenSans-Regular.ttf
+fonts/OpenSans-Semibold.ttf
+fonts/fontawesome_webfont.ttf
+form-validation/formset
+form-validation/formset/form/field/arg
+form-validation/formset/form/field/arg0
+form-validation/formset/form/field/arg1
+form-validation/formset/form/field/arg2
+form-validation/formset/form/field/arg3
+form-validation/global
+fpx/bank_statuses
+friends/ids.json
+friends/ids.json?cursor=
+friends/ids.json?user_id=
+friends/list.json
+friends/list.json?user_id=
+friendships/create.json
+friendships/create.json?user_id=
+friendships/destroy.json
+friendships/destroy.json?user_id=
+friendships/incoming.json?cursor=
+friendships/lookup.json?screen_name=
+friendships/lookup.json?user_id=
+friendships/no_retweets/ids.json
+friendships/outgoing.json?cursor=
+friendships/show.json
+friendships/update.json
+g_m\u001cfq\u001fnvno
```

Comment:
This looks like noise

Reply:
I don't think line 1126 is noise but probably unique to that application. Will remove